docs/tutorials/overwrite-player-at-runtime.md

```diff
@@ -0,0 +1,154 @@
+- Overwrite default inline css shaka-player.
+
+In this tutorial, we will guide you to: add a custom button, modify css style,
+modify a function, detach/add new event listener to an existing element,
+(in practice you could even rewrite shaka-player a whole at runtime).
+
+#### Add a button
+When using shaka player from CDN, your app will have access to global `shaka` variable
+(make sure you don't use global variable `shaka` anywhere in your app,
+we could accidentally overwrite each other).
+
+First let take a look at these button (don't be afraid):
+- [Cast](https://github.com/google/shaka-player/blob/master/ui/cast_button.js)
+- [Fast forward](https://github.com/google/shaka-player/blob/master/ui/fast_forward_button.js)
+- [Play](https://github.com/google/shaka-player/blob/master/ui/play_button.js)
+- [Air play](https://github.com/google/shaka-player/blob/master/ui/airplay_button.js)
+
+What do you see in common ?
+A structure compose of 3 parts: a button extends of basic [element](https://github.com/google/shaka-player/blob/master/ui/element.js) attached to shaka.ui namespace
+A Factory class and a register method for that Factory class.
+
+Let copy that and put it right after fullscreen button!
+
+```js
+shaka.ui.SkipButton = class extends shaka.ui.Element {
+  constructor(parent, controls) {
+    super(parent, controls);
+
+    // The actual button that will be displayed
+    this.button_ = document.createElement('button');
+    this.button_.textContent = 'Skip current video';
+    // Screen readers will read "skip button"
+    this.button_.setAttribute('aria-label', 'Skip');
+    this.parent.appendChild(this.button_);
+  }
+};
+
+shaka.ui.SkipButton.Factory = class {
+  create(rootElement, controls) {
+    return new shaka.ui.SkipButton(rootElement, controls);
+  }
+};
+
+shaka.ui.Controls.registerElement(
+    'skip', new shaka.ui.SkipButton.Factory());
+```
+
+That basically it! When you init shaka-player, make sure you call that `skip` button out
+
+```js
+// init config shaka-player section
+config['controlPanelElements'] = [
+  'play_pause', 'mute', 'volume', 'time_and_duration', 'spacer',
+  'overflow_menu', 'fullscreen', 'skip' // order matter, we put skip behind fullscreen
+]
+```
+
+#### Modify html
+Default in shaka-player we have our seeker bar *after* controls bar, what if we want it before ?
+
+```js
+var shakaSeekBarContainer = document.getElementsByClassName('shaka-seek-bar-container')[0]
+var shakaBottomControls = document.getElementsByClassName('shaka-bottom-controls')[0]
+shakaBottomControls.insertBefore(shakaSeekBarContainer, shakaBottomControls.firstChild)
+```
+
+The player is just javascript, html and css, you could change it however you like!
+
+
+#### Modify css style
+Some default styling not fit with the theme of your site ? We could easily fix that!
+
+There are some methods we could use to fix our styling, namely:
+- using css `!important` attribute. (it's perfectly fine)
```

Comment:
Using `!important` as a hack to fix CSS errors is also a bad idea.  There are better way of fixing problems, like making sure your CSS is loaded after ours so yours replaces ours.  If there is a problem that you can't fix, file a bug and we can see about fixing it the right way.